libs/probation-integration-commons/src/main/kotlin/uk/gov/justice/digital/hmpps/message/SimpleHmppsEvent.kt

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.digital.hmpps.message
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter
+import com.fasterxml.jackson.annotation.JsonAnySetter
+import java.time.ZonedDateTime
+
+interface HmppsEvent {
+    val eventType: String
+}
+
+data class SimpleHmppsEvent(
```

Comment:
Minor thing - are we ever going to have a "complex" domain event and if not, should we drop the Simple prefix?